tests/Rules/Validation/ScopeIlluminateRequestValidateMethodTest.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Rules\Validation;
+
+use Hihaho\PhpstanRules\Rules\Validation\ScopeIlluminateRequestValidateMethods;
+use PHPStan\Analyser\Error;
+use PHPStan\Rules\Rule;
+use PHPStan\Testing\RuleTestCase;
+use PHPUnit\Framework\Attributes\Test;
+
+/**
+ * @extends RuleTestCase<ScopeIlluminateRequestValidateMethods>
+ */
+final class ScopeIlluminateRequestValidateMethodTest extends RuleTestCase
+{
+    protected function getRule(): Rule
+    {
+        return new ScopeIlluminateRequestValidateMethods();
+    }
+
+    #[Test]
+    public function illuminate_http_request_does_not_use_unvalidated_methods_outside_app_http_requests_namespace(): void
+    {
+        /** @var Error[] $errors */
+        $errors = $this->gatherAnalyserErrors([__DIR__ . '/../../stubs/App/Http/Controllers/PeopleControllerStub.php']);
+        $validErrors = array_filter(
```

Comment:
there are more errors present, but we only want to assert the errors from the rules we are introducing into this PR